src/components/geojson-map.vue

```diff
@@ -518,14 +543,79 @@ defineExpose({
     padding-inline: 8px;
     user-select: none;
   }
+
+  .control-bar {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    top: $geojson-map-spacing;
+    right: $geojson-map-spacing;
+    z-index: $geojson-map-z-index;
+    gap: 4px;
+
+    button {
+      background: $geojson-map-background-color;
+      padding: 8px;
+      border-radius: $geojson-map-radius;
+      border: 1px solid $geojson-map-border-color;
+      cursor: pointer;
+      -webkit-tap-highlight-color: transparent;
+
+      &:hover {
+      background: $geojson-map-muted-background-color;
+      }
+
+      .fitToAllFeaturesIcon {
+        width: 20px;
+        height: 20px;
+      }
+    }
+  }
+
+  .ol-zoom {
+    position: absolute;
+    right: $geojson-map-spacing;
+    bottom: $geojson-map-spacing;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    align-items: center;
+    box-shadow: none;
+    background: $geojson-map-background-color;
+    border-radius: $geojson-map-radius;
+    overflow: hidden;
+    border: 1px solid $geojson-map-border-color;
+
+    button,
+    button:hover,
+    button:focus,
+    button:active {
+      height: 37px;
+      width: 36px;
+      border: none;
+      background: $geojson-map-background-color;
+      -webkit-tap-highlight-color: transparent;
+      font-size: 24px;
+      font-weight: 300;
+      cursor: pointer;
+      &:first-child {
+        border-bottom: 1px solid $geojson-map-border-color;
+      }
+      &:hover {
+        background: $geojson-map-muted-background-color;
+      }
+    }
+  }
 }
 </style>
 
 <i18n lang="json5">
 {
   "en": {
     // {count} and {total} are both numbers.
-    "showing": "Showing {count} of {total}"
+    "showing": "Showing {count} of {total}",
+    // Shown above control button on map to zoom out to show all features
+    "zoomToFit": "Zoom to fit all options"
```

Comment:
Even though the features are clickable, I don't really think of them as "options". Is this text from Web Forms? I think they're more so options in Web Forms. What do you think of "Zoom to fit all data"?

Reply:
Changed in Done in https://github.com/getodk/central-frontend/pull/1366